tensorflow_serving/core/basic_manager.cc

```diff
@@ -230,8 +230,8 @@ Status BasicManager::Create(Options options,
   return Status::OK();
 }
 
-BasicManager::BasicManager(Env* const env, const uint32 num_load_threads,
```

Comment:
why do we need this change?

Reply:
I don’t quite understand why max_num_load_retries is not modified by const, so in order to be consistent, I made changes, which may not be appropriate.

Reply:
i'd recommend reverting this file, as i dont see it related to the change you are doing. can you please revert this file?

also can you please update the commit message to read better -- mention the new flags that you have added, and what they do.

thanks!

Reply:
ok, I've completed.